wicket-util/src/test/java/org/apache/wicket/util/crypt/AESCryptTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.wicket.util.crypt;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.Test;
+
+import java.security.GeneralSecurityException;
+
+import javax.crypto.SecretKey;
+
+public class AESCryptTest
+{
+	@Test
+	public void encrypDecrypt() throws GeneralSecurityException
+	{
+		boolean unlimitedStrengthJurisdictionPolicyInstalled = SunJceCryptTest
+			.isUnlimitedStrengthJurisdictionPolicyInstalled();
+		Assumptions.assumeTrue(unlimitedStrengthJurisdictionPolicyInstalled);
+
+		SecretKey secretKey = CipherUtils.generatePBEKey(
+			"myWeakPassword", "PBKDF2WithHmacSHA1", "AES", 
+			CipherUtils.randomByteArray(16), 65536, 256);
+		
+		AbstractJceCrypt crypt = new AESCrypt(secretKey);
+
+		String input1 = "input1";
+		String encrypted = crypt.encryptUrlSafe(input1);
+
+		String input2 = "input2";
```

Comment:
let's use some UTF-8 characters here

Reply:
We already use it inside AbstractJceCrypt and AESCrypt.